frontend/ui/widget/openwithdialog.lua

```diff
@@ -70,17 +71,15 @@ function OpenWithDialog:init()
             end
         end,
 
-        width = math.floor(self.width * 0.9),
-        max_width = math.floor(self.width * 0.9 - 2 * Size.border.window),
-        height = self.height,
```

Comment:
Trusting you that these height= and width= are not needed :)

Reply:
Yes, we do not pass height to any checkbutton, and we have max_width as width.